plugins/Installation/FormFirstWebsiteSetup.php

```diff
@@ -37,6 +39,13 @@ function init()
         $timezones = API::getInstance()->getTimezonesList();
         $timezones = array_merge(array('No timezone' => Piwik::translate('SitesManager_SelectACity')), $timezones);
 
+        // Use server timezone as default. If server timezone is UTC, it is likely
+        // a default not specified explicitly by the sysadm, so ignore this.
+        $timezone = PIWIK_DEFAULT_TIMEZONE;
+        if (in_array($timezone, array('UTC', 'Etc/UTC', 'GMT', 'Etc/GMT'))) {
```

Comment:
is it worth it to compare this all `Lowercase`? Or maybe they cannot be set in different cases?

Reply:
Good point. It seems that PHP does indeed treat timezones as case-insensitive. I didn't know that.